supermercado/scripts/cli.py

```diff
@@ -42,22 +46,43 @@ def union(inputtiles, parsenames):
     for u in unioned:
         click.echo(json.dumps(u))
 
-cli.add_command(union)
 
+class zoomCustomType(click.ParamType):
+    """Custom zoom type."""
+
+    name = "zoom"
+
+    def convert(self, value, param, ctx):
+        """Validate and parse band index."""
+        try:
+            assert re.match(r"^[0-9]+(..[0-9]+)?$", value)
+            zooms = list(map(int, value.split("..")))
+            assert all(z > 0 for z in zooms)
+            return min(zooms), max(zooms)
+
+        except (AssertionError):
+            raise click.ClickException(
+                "zoom must be a integer or a 'min..max' string "
+                "representing a zoom range, e.g. 9..12"
+            )
 
-@click.command('burn')
+
+@cli.command('burn')
 @cligj.features_in_arg
 @cligj.sequence_opt
-@click.argument('zoom', type=int)
+@click.argument(
+    'zoom',
+    type=zoomCustomType()
+)
 def burn(features, sequence, zoom):
     """
     Burn a stream of GeoJSONs into a output stream of the tiles they intersect for a given zoom.
     """
     features = [f for f in super_utils.filter_polygons(features)]
 
-    tiles = burntiles.burn(features, zoom)
-    for t in tiles:
-        click.echo(t.tolist())
+    tiles = (
+        burntiles.burn(features, zoom) for zoom in range(zoom[0], zoom[1] + 1)
```

Comment:
We don't have to re-burn for every zoom; we can use the maxzoom to generate the tile-cover, then:
```python
tiles[:, :2] >>= 1
tiles[:, 2] -= 1
np.unique(tiles, axis=0)
```
This will output the parents of `tiles`. cc @pratikyadav

Reply:
yeah makes sense 👍

Reply:
Explain @dnomadb ? Does that not risk false positives, sweeping in small child tiles that are outside the features?

Reply:
@sgillies y of course!

A feature that covers a tile at zoom z will always cover said tiles at zooms z-n. False positives would be a (huge) risk going to _higher_ zooms but if we perform the tile cover at the _maxzoom_ of the provided range, we can then derive lower zooms directly from this array.

Reply:
Right! Thanks, I must have had some min/max zoom confusion in my head at the end of the day.